File: Naloga 20/naloga_20.py
def naloga_20():
    stevilo = str()  # samo da obstaja
    while True:
        stevilo = (input())
        if stevilo == "d":
            break
        if int(stevilo) % 2 == 0:
            print("sodo")
        else:
            print("liho")
if __name__ == '__main__':
    naloga_20()

File: Naloga 20/naloga_20.cpp
#include <iostream>

bool deljivost(int x, int y) {
    if ((x % y) == 0) {
        return true;
    }
    return false;
};

void naloga_20() {
    int stevilo;
    do {
        std::cin >> stevilo;
        if (stevilo == 100) {break;};
        if (deljivost(stevilo, 2)) {
            std::cout << "sodo" << std::endl;
        }
        else {
             std::cout << "liho" << std::endl;
        };
    }while (stevilo != 100);
};
int main () {
    naloga_20();
    return 0;
};